plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py

```diff
@@ -32,32 +64,114 @@ def uiCreate(cls, parentLayout):
         topForm = cmds.columnLayout('actionOptionsForm', rowSpacing=5)
 
         kFileOptionsStr = getMayaUsdString("kFileOptions")
-        kMakePathRelativeStr = getMayaUsdString("kMakePathRelativeToSceneFile")
-        kMakePathRelativeAnnStr = getMayaUsdString("kMakePathRelativeToSceneFileAnn")
+        kMakePathRelativeStr = getMayaUsdString("kMakePathRelativeTo" + relativeToWhat)
+        kMakePathRelativeAnnStr = getMayaUsdString("kMakePathRelativeTo" + relativeToWhat + "Ann")
  
         optBoxMarginWidth = mel.eval('global int $gOptionBoxTemplateDescriptionMarginWidth; $gOptionBoxTemplateDescriptionMarginWidth += 0')
         cmds.setParent(topForm)
         cmds.frameLayout(label=kFileOptionsStr, collapsable=False)
         widgetColumn = cmds.columnLayout()
-        cmds.checkBox(cls.kMakePathRelativeCheckBox, label=kMakePathRelativeStr, ann=kMakePathRelativeAnnStr)
+        cmds.checkBox(cls.kMakePathRelativeCheckBox + relativeToWhat, label=kMakePathRelativeStr, ann=kMakePathRelativeAnnStr)
 
     @classmethod
-    def uiInit(cls, parentLayout, filterType):
+    def uiInit(cls, parentLayout, canBeRelative, relativeToWhat):
+        """
+        Helper method to initialize the UI layout for the file relative actions.
+
+        Input parentLayout arg is expected to the a scroll layout into which controls
+        can be added.
```

Comment:
Sorry I realize I made a grammar mistake when I added this. Can you fix it. 
```suggestion
        Input parentLayout arg is expected to be a scroll layout into which controls
        can be added.
```